src/index.js

```diff
@@ -4,9 +4,29 @@ import menu from './menu';
 import contact from './contact';
 
 const parent = document.getElementById('content');
-parent.style.background = "url('assets/img/b.jpg') center";
+parent.style.background = "url('assets/img/b.jpg') center no-repeat";
 
 parent.appendChild(nav());
-parent.appendChild(home());
-parent.appendChild(menu());
-parent.appendChild(contact());
+const link = document.querySelectorAll('.links');
+const main = document.createElement('div');
+
+main.appendChild(home());
+parent.appendChild(main);
+
+const tab = (lid) => {
+  let a;
+  if (lid === 'home') a = home();
+  if (lid === 'menu') a = menu();
+  if (lid === 'contact') a = contact();
+
+  return a;
```

Comment:
[OPTIONAL] I think you could use more descriptive variable names here.

Reply:
done